### ISSUE TITLE
Released version 3.2.0 has wrong version

### ISSUE THREAD

**Issue by peterM** [Pro/dkim-exchange#261]
I downloaded version 3.2.0 but after installation it say i have version 3.1.0

**Comment by robatwork** [comment]
/me too

In Information tab it shows 3.1.0
In About tab it shows 3.2.0

So you probably DO have 3.2.0, someone forgot to update the info tab

**Comment by peterM** [comment]
I do not understand why it happen because version is loaded by code:
```csharp
oDkimSignerInstalled = Version.Parse(FileVersionInfo.GetVersionInfo(Path.Combine(Constants.DkimSignerPath, Constants.DkimSignerAgentDll)).ProductVersion);
```

And Assembly version for both projects is set to: 

```csharp
[assembly: AssemblyVersion("3.2.0")]
[assembly: AssemblyFileVersion("3.2.0")]
```

Also file attributes are correct:
![image](https://user-images.githubusercontent.com/2149830/57010142-6f502f00-6bfb-11e9-9ec3-e97a4cdb6ea0.png)

**Comment by peterM** [comment]
There is possibility to do offline installation but point 6 is at least to me a bit mistery:

![image](https://user-images.githubusercontent.com/2149830/57010995-d2dc5b80-6bff-11e9-9860-f5f7369fdc26.png)

From where it is possible to down,load this `zip` file ? I did not find

**Comment by peterM** [comment]
Manual FIX for v3.2.0:

**Before you start make sure you have backup**

1.) Open configurator
2.) stop service
3.) close configurator
4.) download zip: [https://github.com/Pro/dkim-exchange/archive/v3.2.0.zip](https://github.com/Pro/dkim-exchange/archive/v3.2.0.zip)
5.) extract zip to some folder
6.) Check version of Exchange what you have installed
7.) go to path `PathOfZipExtracted\dkim-exchange-3.2.0\Src\Exchange.DkimSigner\bin\{Version of your exchange}\
8.) copy all files to folder (and replace old) where is installed `dkim signer`
9.) start configurator
10.) you should see `installed version 3.2.0`
11.) start service
12.) close configurator

**Comment by sarthurdev** [comment]
Is this resolved in the new pre-release of v3.2.0? Did not face this issue in my testing.
